docs/schema/customaction-element-customactiondefinitions-complextypeapphostwebfeatures.md

```diff
@@ -17,12 +18,11 @@ ms.assetid: cb7b6514-f038-eb1a-88a8-9515425b150e
 > The string `app` appears as part of or all of some element, attribute, and file names because SharePoint Add-ins were originally called "apps for SharePoint." To ensure backward compatibility, the schemas have not been changed. 
 
 ## Element information
+**Element type**: CustomActionDefinition
```

Comment:
We've been directed to keep Markdown tables as tables. You can use "Information/Location" or "Name/Value" as headers, or if you find a more specific header that fits, use that.